crates/sui-transactional-test-runner/src/test_adapter.rs

```diff
@@ -1048,6 +1051,56 @@ impl<'a> SuiTestAdapter<'a> {
         Ok(res)
     }
 
+    fn named_variables(&self) -> BTreeMap<String, String> {
+        let mut variables = BTreeMap::new();
+        let named_addrs = self
+            .compiled_state
+            .named_address_mapping
+            .iter()
+            .map(|(name, addr)| (name.clone(), format!("{:#02x}", addr)));
+
+        let objects = self
+            .object_enumeration
+            .iter()
+            .filter_map(|(oid, fid)| match fid {
```

Comment:
Not sure I understand this part. What is a `FakeID`?

Reply:
This is basically lifted from @oxade 's fn to create graphql variables. This is what allows us to refer to objects by object(2,0) instead of the actual objectId